playground/mocks/data/idp/idx/authenticator-verification-custom-app-push-reject.json

```diff
@@ -0,0 +1,280 @@
+          },
+          {
+            "name":"stateHandle",
+            "required":true,
+            "value":"02jScqx_dpaclG7Hury1J4J2qS2B_bltQte8eCSQUe",
+            "visible":false,
+            "mutable":false
+          }
+        ],
+        "accepts":"application/json; okta-version=1.0.0"
+      },
+      {
+        "rel":[
+          "create-form"
+        ],
+        "name":"select-authenticator-authenticate",
+        "href":"http://localhost:3000/idp/idx/challenge",
+        "method":"POST",
+        "produces":"application/ion+json; okta-version=1.0.0",
+        "value":[
+          {
+            "name":"authenticator",
+            "type":"object",
+            "options":[
+              {
+                "label":"Wells Fargo Verify App",
+                "value":{
+                  "form":{
+                    "value":[
+                      {
+                        "name":"id",
+                        "required":true,
+                        "value":"aut198w4v0f8dr8gT0g4",
+                        "mutable":false
+                      },
+                      {
+                        "name":"methodType",
+                        "type":"string",
+                        "required":false,
+                        "options":[
+                          {
+                            "label":"Get a push notification",
+                            "value":"push"
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                },
+                "relatesTo":"$.authenticators.value[1]"
+              },
+              {
+                "label":"Duo Security",
+                "value":{
+                  "form":{
+                    "value":[
+                      {
+                        "name":"id",
+                        "required":true,
+                        "value":"aut1bnark0PdN44oq0g4",
+                        "mutable":false
+                      },
+                      {
+                        "name":"methodType",
+                        "required":false,
+                        "value":"duo",
+                        "mutable":false
+                      }
+                    ]
+                  }
+                },
+                "relatesTo":"$.authenticatorEnrollments.value[0]"
+              }
+            ]
+          },
+          {
+            "name":"stateHandle",
+            "required":true,
+            "value":"02jScqx_dpaclG7Hury1J4J2qS2B_bltQte8eCSQUe",
+            "visible":false,
+            "mutable":false
+          }
+        ],
+        "accepts":"application/json; okta-version=1.0.0"
+      }
+    ]
+  },
+  "messages":{
+    "type":"array",
+    "value":[
+      {
+        "message":"You have chosen to reject this login.",
+        "i18n":{
+          "key":"api.authn.poll.error.push_rejected",
+          "params":[
+            
+          ]
+        },
+        "class":"ERROR"
```

Comment:
As part of cleanup.. If this message needs to be displayed as text please change this to `INFO`

Reply:
This needs to be an error. I fixed the code to display it as an error banner by overriding ShowMessages.